2024/07/04 08:17:31 [error] 15#15: *2 connect() failed (113: Host is unreachable) while connecting to upstream, client: 192.168.65.1, server: esignature.test, request: "POST /api/v1/signup HTTP/1.1", upstream: "fastcgi://172.25.0.5:9000", host: "localhost:8088"
2024/07/04 08:19:02 [error] 13#13: *2 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.65.1, server: esignature.test, request: "POST /api/v1/signup HTTP/1.1", upstream: "fastcgi://172.25.0.5:9000", host: "localhost:8088"
2024/07/04 08:19:05 [error] 13#13: *2 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.65.1, server: esignature.test, request: "POST /api/v1/signup HTTP/1.1", upstream: "fastcgi://172.25.0.5:9000", host: "localhost:8088"
